### PR TITLE
release fiat-crypto and deps

### PR DESCRIPTION
**Pull request by andres-erbsen**
I ran the script at https://github.com/coq/opam/pull/2992 and I don't really understand what I am doing.

@JasonGross please review

### CODE REVIEW

File: released/packages/coq-bedrock2/coq-bedrock2.0.0.8/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+]
+install: [make "EXTERNAL_DEPENDENCIES=1" "install_bedrock2"]
+run-test: [
+  [make "-j%{jobs}%" "-C" "bedrock2" "EXTERNAL_DEPENDENCIES=1" "test"]
+]
+depends: [
+  "conf-findutils" {build}
+  "conf-python-3" {build & with-test}
+  "coq" {>= "8.18~"}
+  "coq-coqutil" {= "0.0.5"}
```
Comment (JasonGross) [inline released/packages/coq-bedrock2/coq-bedrock2.0.0.8/opam:22]:
```suggestion
  "coq-coqutil" {= "0.0.6"}
```

File: released/packages/coq-fiat-crypto/coq-fiat-crypto.0.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ["etc/test-run-fiat-crypto-silent.sh" "fiat_crypto"] {with-test}
+]
+depends: [
+  "conf-findutils" {build}
+  "ocaml" {build & >= "4.08~"}
+  "ocamlfind" {build}
+  "coq" {>= "8.18~"}
+  "coq-coqprime" {>= "1.2.0"}
+  "coq-rewriter" {>= "0.0.6"}
+  "coq-rupicola" {= "0.0.9"}
```
Comment (JasonGross) [inline released/packages/coq-fiat-crypto/coq-fiat-crypto.0.1.3/opam:28]:
```suggestion
  "coq-rupicola" {= "0.0.10"}
```

File: released/packages/coq-rupicola/coq-rupicola.0.0.10/opam [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+bug-reports: "https://github.com/mit-plv/rupicola/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%" "EXTERNAL_DEPENDENCIES=1" "all"]
+]
+install: [make "EXTERNAL_DEPENDENCIES=1" "install"]
+depends: [
+  "conf-findutils" {build}
+  "coq" {>= "8.18~"}
+  "coq-bedrock2" {= "0.0.7"}
```
Comment (JasonGross) [inline released/packages/coq-rupicola/coq-rupicola.0.0.10/opam:20]:
```suggestion
  "coq-bedrock2" {= "0.0.8"}
```

File: released/packages/coq-fiat-crypto/coq-fiat-crypto.0.1.3/opam [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  "coq-bedrock2-compiler" {= "0.0.7"}
```
Comment (JasonGross) [inline released/packages/coq-fiat-crypto/coq-fiat-crypto.0.1.3/opam:29]:
```suggestion
  "coq-bedrock2-compiler" {= "0.0.7"}
```

**Comment by palmskog** [comment]
We have the following error on Coq 8.15:
```
# File "/builds/coq/opam/opam-root-4.07.1-2.1.2-sandbox/4.07.1/.opam-switch/build/coq-riscv.0.0.5/src/riscv/Spec/Primitives.v", line 136, characters 6-12:
# Error: This command does not support this attribute: global.
# [unsupported-attributes,parsing]
```
Probably some Coq bound needs adjustment.

**Pull request merged**